packages/firestore/test/util/helpers.ts

```diff
@@ -401,6 +404,55 @@ export function docUpdateRemoteEvent(
   return aggregator.createRemoteEvent(doc.version);
 }
 
+export class TestBundledDocuments {
```

Comment:
Can this be inlined? `bundledDocuments` would just return `BundledDocuments`.

Reply:
I added this because it hard to add code to `after(op)` method otherwise, suggestions welcome.